src/test/java/com/hocs/test/glue/MarkUpStepDefs.java

```diff
@@ -119,9 +166,124 @@ public void theTopicShouldBeSetAsTheOrdinalTopic(String ordinal) {
         }
     }
 
+    @Then("^the case should be assigned to the \"([^\"]*)\" for drafting$")
+    public void theCaseShouldBeAssignedToTheDraftTeam(String draftingTeam) {
+        switch (draftingTeam.toUpperCase()) {
+            case "ANIMALS IN SCIENCE REGULATION UNIT" :
+                topics.assertElementTextIs(
+                        topics.autoAssignedDraftTeam, draftingTeam);
+                break;
+            case "PUBLIC PROTECTION UNIT" :
+                topics.assertElementTextIs(
+                        topics.autoAssignedDraftTeam, draftingTeam);
+                break;
+            case "DOMESTIC VIOLENCE PROTECTION ORDERS" :
+                topics.assertElementTextIs(
+                        topics.autoAssignedDraftTeam, draftingTeam);
+                break;
+            default:
+                System.out.println(draftingTeam
+                        + " is not defined within " + getClass().getSimpleName()
+                        + " class, " + getMethodName() + " method");
+                draftingTeam = null;
+                assumeNotNull(draftingTeam);
+        }
+    }
+
+    @Then("^the case should be assigned to the \"([^\"]*)\" for approval$")
+    public void theCaseShouldBeAssignedToThePrivateOfficeTeam(String privateOfficeTeam) {
+        switch (privateOfficeTeam.toUpperCase()) {
+            case "MINISTER FOR LORDS" :
+                topics.assertElementTextIs(topics.autoAssignedPrivateOfficeTeam,
+                        privateOfficeTeam);
+                break;
+            case "UNDER SECRETARY OF STATE FOR CRIME, SAFEGUARDING AND VULNERABILITY" :
+                topics.assertElementTextIs(topics.autoAssignedPrivateOfficeTeam,
+                        privateOfficeTeam);
+                break;
+        }
+    }
+
     @When("^I select a primary topic$")
     public void iSelectAPrimaryTopic() throws Throwable {
         // Write code here that turns the phrase above into concrete actions
         throw new PendingException();
     }
+
+    @Then("^the case should be found in the \"([^\"]*)\" team$")
+    public void theCaseShouldBeFoundInTheTeamTeam(String team) {
+        homepage.goHome();
+        switch (team.toUpperCase()) {
+            case "PUBLIC PROTECTION UNIT" :
+                homepage.selectPublicProtectionUnit();
+                workstacks.assertCaseReferenceIsVisible();
+                break;
+            case "WINDRUSH COMPENSATION TEAM" :
+                //homepage.selectWindrusCompensationTeam();
```

Comment:
I'd recommend a `pendingStep` here. Otherwise, this step will `pass` as it has nothing to do, but no failure condition. `pendingStep` will show it correctly as incomplete.